To-do list/Models/Helpers/AuthApiHelper.swift

```diff
@@ -27,7 +27,8 @@ final class AuthApiHelper {
     
     func executeLoginRequest (email: String,
                               password: String,
-                              completion: @escaping (LoginResponse?) -> Void ) {
+                              completion: @escaping (Result <LoginResponse,
```

Comment:
Don't break down arguments of Result<> struct. Declare it on the single line instead.